components/Footer.js

```diff
@@ -0,0 +1,27 @@
+import Cards from "./Cards";
+
+const Footer = () => {
+  return (
+    <footer>
+      <div>
+        <Cards />
+      </div>
+      <a
+        href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        Powered by Vercel
+      </a>
+      <a
+        href="https://www.flaticon.com/home"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        Icons by Flaticon
+      </a>
+    </footer>
+  );
+};
+
+export default Footer;
```

Comment:
same question about adding a line at end of file here.

Reply:
I updated the settings in my IDE to add newline on save.